test/topo/topotest.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package topo
+
+import (
+	"context"
+	"github.com/onosproject/onos-api/go/onos/topo"
+	"github.com/onosproject/onos-config/test/utils/gnmi"
+	toposdk "github.com/onosproject/onos-ric-sdk-go/pkg/topo"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+// NewTopoClient creates a new topo SDK client
+func NewTopoClient() (toposdk.Client, error) {
```

Comment:
I am not we need a different suite for this test. Lets have a common util functions for topo like e2t that we can use it for other tests as well.  We can add this test to gnmi suite. Perhaps gnmi should be renamed to config tests to include tests related to Set/Get operations + topo integration.

Reply:
technically, this test tests connectivity in SB of onos-config + topo integration so we testing core of onos-config